tests/unit/components/form/RenameFileForm.spec.js

```diff
@@ -0,0 +1,103 @@
+import { installQuasarPlugin } from '@quasar/quasar-app-extension-testing-unit-jest';
+import { shallowMount } from '@vue/test-utils';
+import RenameFileForm from 'src/components/form/RenameFileForm';
+import { Notify } from 'quasar';
+
+installQuasarPlugin({
+  plugins: [Notify],
+});
+
+jest.mock('vue-i18n', () => ({
+  useI18n: () => ({
+    t: (t) => t,
+  }),
+}));
+
+jest.mock('src/composables/Project', () => ({
+  gitRemove: jest.fn(() => Promise.resolve()),
+  readDir: jest.fn(() => ['test']),
+  rename: jest.fn(
+    (old, rename) => (rename === 'projectName/rename' ? Promise.resolve() : Promise.reject()),
+  ),
+}));
+
+describe('Test component: RenameFileForm', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallowMount(RenameFileForm, {
+      props: {
+        projectName: 'projectName',
+        file: {
+          id: 'test',
+          isfolder: false,
+          label: 'test',
+        },
+      },
+    });
+  });
+
+  describe('Test variables initialization', () => {
+    describe('Test props: projectName', () => {
+      it('should match "projectName"', () => {
+        expect(wrapper.vm.projectName).toEqual('projectName');
+      });
+    });
+
+    describe('Test props: file', () => {
+      it('should match file', () => {
+        expect(wrapper.vm.file).toEqual({
+          id: 'test',
+          isfolder: false,
+          label: 'test',
+        });
+      });
+    });
+
+    describe('Test variable: fileName', () => {
+      it('should match <empty string>', () => {
+        expect(wrapper.vm.fileName).toEqual('');
+      });
+    });
+
+    describe('Test variable: submitting', () => {
+      it('should be false', () => {
+        expect(wrapper.vm.submitting).toEqual(false);
+      });
+    });
+
+    describe('Test variable: labels', () => {
+      it('should match []', () => {
+        expect(wrapper.vm.labels).toEqual(['test']);
```

Comment:
` should match []` but `toEqual(['test'])` ?

Reply:
done